languages/golang/spammer.go

```diff
@@ -2,33 +2,45 @@ package main
 
 import (
 	"fmt"
+	"log"
 	"time"
 
+	"github.com/DataDog/datadog-go/statsd"
 	"gopkg.in/DataDog/dd-trace-go.v1/ddtrace/tracer"
 )
 
-func makenestedspan() {
-	span := tracer.StartSpan("nested-spam")
-	time.Sleep(1 * time.Millisecond)
-	defer span.Finish()
-}
-
-func makespan() {
-	span := tracer.StartSpan("spam", tracer.ResourceName("spammer"))
-	makenestedspan()
-	defer span.Finish()
-}
-
 func main() {
 	fmt.Printf("Waiting 10 seconds for agent to be ready\n")
 	time.Sleep(10 * time.Second)
 	fmt.Printf("Starting spammer at: %v\n", time.Now().Unix())
 	tracer.Start()
-
+	defer tracer.Stop()
+	statsdClient, err := statsd.New("observer:8125")
+	if err != nil {
+		log.Fatal(err)
+	}
+	stressTest := &stressTest{statsdClient}
 	for {
-		makespan()
+		stressTest.createTrace()
 	}
-
 	fmt.Printf("Finishing at: %v\n", time.Now().Unix())
-	defer tracer.Stop()
+}
+
+type stressTest struct {
+	statsdClient *statsd.Client
+}
+
+// createTrace creates a trace with two spans,
+// where "nested-spam" is a child of "spam".
+// Each span creation sends an event to DogStatsD to
+// help with throughput analysis.
+func (st *stressTest) createTrace() {
+	st.statsdClient.SimpleEvent("transport_sample.span_created", "")
```

Comment:
```suggestion
	st.statsdClient.Incr("transport_sample.span_created", []string{}, 1)
```

https://pkg.go.dev/github.com/DataDog/datadog-go/v5/statsd#Client.Incr

This is the correct function, but double check I have the parameters correct.

Needs to be the same for below as well